cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperConfiguration.java

```diff
@@ -7,29 +7,35 @@
 
 import io.strimzi.api.kafka.model.ZookeeperClusterSpec;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
 
 /**
  * Class for handling Zookeeper configuration passed by the user
  */
 public class ZookeeperConfiguration extends AbstractConfiguration {
 
     private static final List<String> FORBIDDEN_OPTIONS;
-    private static final Map<String, String> DEFAULTS;
+    protected static final Map<String, String> DEFAULTS;
 
     static {
-        FORBIDDEN_OPTIONS = asList(
-                ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(" *, *"));
-
-        DEFAULTS = new HashMap<>();
-        DEFAULTS.put("tickTime", "2000");
-        DEFAULTS.put("initLimit", "5");
-        DEFAULTS.put("syncLimit", "2");
-        DEFAULTS.put("autopurge.purgeInterval", "1");
+        FORBIDDEN_OPTIONS = new ArrayList<>();
+        FORBIDDEN_OPTIONS.addAll(Arrays.asList(ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(" *, *")));
+        FORBIDDEN_OPTIONS.add("zookeeper.snapshot.trust.empty");
```

Comment:
You said that this option is ignored in config file and used only from system property. So is this really needed?